sopel/modules/wiktionary.py

```diff
@@ -52,7 +54,9 @@ def wikt(word):
             mode = 'particle'
         elif 'id="Preposition"' in line:
             mode = 'preposition'
-        elif 'id="' in line:
+        # 'id="' can occur in definition lines <li> when <sup> tag is used for references;
+        # make sure those are not excluded (see e.g., abecedarian).
+        elif ('id="' in line) and ('<li>' not in line):
```

Comment:
Could some `<li>` elements contain classes, IDs, etc. that would make this match fail? Is there any reason you've found in testing not to use `<li` without the closing `>`?

Reply:
Good question; I was really just building off the previous logic, which, as implemented, assumes that a definition line will contain a plain `<li>` tag with no attributes or properties. However, I don't know whether this is always true  for definitions but seems to have worked for this long and I didn't want to change too much.

It is possible that `<li>` tags with classes, IDs, etc. are found elsewhere in the page, but are not part of the definition, and expanding the match to allow those may introduce more problems in the current parsing logic. 

Granted, my list of test words is very small and likely doesn't even come close to edge cases, it may be best to leave as is until another failing case is found, because this match seems to work for now (and has been). I think words are more likely to fail because of whats between the `<li>` tags (like the `<sup...>` in this case) than the `<li>` match itself.

Reply:
Fair enough. Let's let this sleeping dog lie. :grinning: